src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { PreparedTodo } from '../types/preparedTodo';
+
+type Props = {
+  todo: PreparedTodo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
you can return without using word `return` , just in round brackets ({ todo }) => (